builds/misc/templates/build-packages.yaml

```diff
@@ -302,7 +306,7 @@ stages:
           # build iot-identity-service here since they cannot build native aarch64 packages
           - bash: |
               set -ex
-              git clone --recurse-submodules https://github.com/Azure/iot-identity-service.git -b release/1.2
+              git clone --recurse-submodules https://github.com/Azure/iot-identity-service.git -b release/1.3
```

Comment:
I don't think we should use the IIS from `release/1.3` branch since this is for iotedge's `main` branch. 
Use the `main` branch of IIS please.

Reply:
816f87065709e2f8ef63890e4add5212d9dc8f83